test/IntegrationTests/MSTest.Acceptance.IntegrationTests/MSTest.Acceptance.IntegrationTests.csproj

```diff
@@ -38,4 +43,8 @@
     </None>
   </ItemGroup>
 
+  <Target Name="CopyNuGetPackagesForTestAssets" BeforeTargets="BeforeBuild">
+    <Copy SourceFiles="$(NuGetPackageRoot)aspire.hosting.testing\$(AspireHostingTestingVersion)\aspire.hosting.testing.$(AspireHostingTestingVersion).nupkg" DestinationFiles="$(ArtifactsTmpDir)/packages/aspire.hosting.testing.$(AspireHostingTestingVersion).nupkg" />
```

Comment:
SkipUnchangedFiles?

https://learn.microsoft.com/en-us/visualstudio/msbuild/copy-task?view=vs-2022

Reply:
Will do as a follow-up (right now) as I can apply the same pattern in some other projects